modules/encrypt-node/test/encrypt.test.ts

```diff
@@ -370,6 +370,34 @@ describe('encrypt structural testing', () => {
     // This will throw if it does not deserialize correctly
     deserializeSignature(footerSection)
   })
+
+  it('can encrypt empty message', async () => {
```

Comment:
nit: any reason to check the encryption context and EDKs in the browser version of this test but not the Node version?

Reply:
Just a miss, inc

Reply:
to elaborate more, when encrypting 0 bytes, the only part of the message that is relevant _is_ the encryption context :)